pywr/domains/river.py

```diff
@@ -170,17 +170,23 @@ class RiverSplitWithGauge(RiverSplit):
         The identifiers to refer to the slots when connect from this Node. Length must be one more than
          the number of extra slots required.
     """
-    def __init__(self, *args, **kwargs):
+    def __init__(self, model, name, mrf=0.0, cost=0.0, mrf_cost=0.0, **kwargs):
         """Initialise a new RiverSplit instance
```

Comment:
I can't remember what the rule for this is although I remember talking about it with you (and you being correct and me being mistaken).